examples/user_guide/Custom_Components.ipynb

```diff
@@ -540,9 +550,22 @@
   }
  ],
  "metadata": {
+  "kernelspec": {
```

Comment:
Please tell me if I can do something to keep these meta data changes out. Thanks.

Reply:
On Unix based systems I use:

```
#!/bin/bash
jq --sort-keys --indent 1 \
    '
    (.cells[] | select(has("outputs")) | .outputs) = []
    | (.cells[] | select(has("execution_count")) | .execution_count) = null
    | .metadata = {"language_info": {"name":"python", "pygments_lexer": "ipython3"}}
    | .cells[].metadata = {}
    | del(.cells[].id)
    ' "$1" > /tmp/$(basename $1) && cat /tmp/$(basename $1) > "$1"
```

to clear out the metadata.